cookbooks/aws-parallelcluster-install/resources/ec2_udev_rules/partial/_common_udev_configuration.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+#
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+action :create_common_udev_files do
+  template 'ec2-volid.rules' do
```

Comment:
Thought: at some point we need to test this stuff actually works.

Reply:
Testing udev rules is not straight forward and requires some effort, I'd not do it in the scope of this PR

Reply:
Absolutely not. "At some point".